src/components/PeopleAndRoles/PeopleAndRoles.vue

```diff
@@ -35,7 +35,7 @@
         outlined
         color="primary"
         :disabled="showOrgPersonForm"
-        @click="addOrgPerson([], IncorporatorTypes.PERSON)"
+        @click="initAdd([], IncorporatorTypes.PERSON)"
```

Comment:
Method renamed to be clear that this intializes the add process, while `performAddEdit` actually performs the change.